Makefile

```diff
@@ -66,26 +67,29 @@ LOCAL_REGISTRY ?= registry:5000
 
 export MANIFESTS_DIR ?= build/_output/manifests
 
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true"
+
 all: check handler
 
 check: vet whitespace-check gofmt-check
 
 format: whitespace-format gofmt
 
 vet: $(GO)
-	$(GO) vet ./cmd/... ./pkg/... ./test/...
+	$(GO) vet ./...
 
 whitespace-format:
 	hack/whitespace.sh format
 
 gofmt: $(GO)
-	$(GOFMT) -w cmd/ pkg/ test/e2e/
+	$(GOFMT) -w *.go test/ hack/ api/ controllers/ pkg/
```

Comment:
There is no *.go file in the root, is there?

Reply:
main.go is not at root level

Reply:
So why do you run *.go here? Should not we target the directory that has main.go?

Reply:
Ahaaa, I missed that. Thanks